synapse/lib/stormlib/storm.py

```diff
@@ -29,6 +36,8 @@ async def _evalStorm(self, text, cast=None):
         text = await s_stormtypes.tostr(text)
         cast = await s_stormtypes.tostr(cast, noneok=True)
 
+        self.runt.snap.core._logStormQuery(text, self.runt.user)
```

Comment:
This doesn't distinguish between being the result of a lib.storm.eval call though?